daemon/graphdriver/overlay2/check.go

```diff
@@ -188,6 +188,11 @@ func usingMetacopy(d string) (bool, error) {
 	// ...and check if the pulled-up copy is marked as metadata-only
 	xattr, err := system.Lgetxattr(filepath.Join(l2, "f"), overlayutils.GetOverlayXattr("metacopy"))
 	if err != nil {
+		// ENOTSUP signifies the FS does not support either xattrs or metacopy. In either case,
+		// it is not a fatal error, and we should report metacopy as unused.
+		if errors.Is(err, unix.ENOTSUP) {
+			return false, nil
```

Comment:
Do you think we need to log this? (I see we do some logging above)

Reply:
I don't think so, because the point of this change is it's a "benign failure." Maybe it would be informative to say the check was "not supported" instead of "not used," but ultimately I think of this check as binary: either we're using metacopy, or we're not.

With that in mind, I conceptualize this as handling a way the OS reports "metacopy not in use" that I overlooked in my first iteration.

Reply:
Yes, was mostly wondering if it would be informative to the user to learn why it's skipping, but I guess we could add in future.

Reply:
I don't mind either way, with a slight bias towards not logging given my reasoning above.